ui/addhardware.ui

```diff
@@ -297,11 +297,6 @@
                                     <property name="visible">True</property>
                                     <property name="can-focus">False</property>
                                     <property name="has-entry">True</property>
-                                    <child internal-child="entry">
-                                      <object class="GtkEntry" id="combobox-entry4">
-                                        <property name="can-focus">True</property>
-                                      </object>
-                                    </child>
```

Comment:
This deletion here shouldn't happen. This is some kind of artifact of using glade. The instances in this patch don't cause runtime problems as is. But when you open the glade file and then re-save it, at least as of glade-3.38.2-5.fc36.x86_64 which I am using, you'll see diffs like:

```diff
@@ -297,6 +297,11 @@
                                     <property name="visible">True</property>
                                     <property name="can-focus">False</property>
                                     <property name="has-entry">True</property>
+                                    <child internal-child="entry">
+                                      <object class="GtkEntry">
+                                        <property name="can-focus">False</property>
+                                      </object>
+                                    </child>
                                   </object>
                                   <packing>
                                     <property name="left-attach">1</property>
```

Notice that when it refills in the entry, can-focus=False, which will screw up interactivity for those affected entries.
You can fix it with

```console
git show > patch.diff
# edit patch.diff and remove everything but those bad diffs
patch -p1 -R < patch.diff
git commit -a --amend
```